src/react-apps/applications/ux-editor/src/containers/EditContainer.tsx

```diff
@@ -102,15 +102,15 @@ class Edit extends React.Component<IEditContainerProps, IEditContainerState> {
       isEditModalOpen: false,
       isItemActive: false,
       isEditMode: false,
-      component: _props.component,
+      component: { ... this.props.component },
```

Comment:
Should there be a whitespace here?
```suggestion
      component: { ...this.props.component },
```

Reply:
Both work fine, so I guess its a personal preference. I have no strong opinions, so Ill go with your suggestion ;)